lib/fathead/mdn_css/parse.pl

```diff
@@ -296,7 +296,7 @@ sub create_redirects {
 
     # Capture content inside parentheses
     # E.g. "::before (:before)"
-    if ( $title =~ m/(::.+) \((:.+)\)/ ) {
+    if ( $title =~ m/(.+) \((:.+|-webkit.+)\)/ ) {
```

Comment:
@rasikapohankar my mistake, those redirects looks unrelated to these changes -- they must have made several changes to MDN recently 👍